src/phrases/phrases-nb.ts

```diff
@@ -838,6 +838,7 @@ const phrases = {
         unSupported: 'Innholdstypen i lenken er ikke støttet',
         newUrlResource: 'Ny ressurs',
         changeUrlResource: 'Rediger ressurs: {{type}}',
+        fullscreen: 'Vis i fullskjerm',
```

Comment:
Marc ville at dette skulle stemme bedre overens med visninga.
```suggestion
        fullscreen: 'Åpne i nytt vindu',
```